gf/test/gf_new_tail_test_mpi.cpp

```diff
@@ -34,7 +34,7 @@ class TailedGreensFunctionMPI : public ::testing::Test
   omega_sigma_gf gf2;
   typedef omega_sigma_gf gf_type;
 
-  TailedGreensFunctionMPI():beta(10), nfreq(10), nsites(5), nspins(2),
+  TailedGreensFunctionMPI():beta(10), nsites(5), nfreq(10), nspins(2),
```

Comment:
what's the rationale of this change? (not that it does anything...)

Reply:
It eliminates warning about different order of declarations in the class and initializers in the ctor.